synapse/tools/storm.py

```diff
@@ -420,6 +615,9 @@ async def main(argv, outp=s_output.stdout):
         async with await s_telepath.openurl(opts.cortex) as proxy:
 
             async with await StormCli.anit(proxy, outp=outp, opts=opts) as cli:
+                completer = StormCompleter(cli)
+                await completer.anit()
+                cli.completer = completer
```

Comment:
Can we move this after the `onecmd` option is handled?
Otherwise we're doing a bunch of work for a non-interactive use case and that's not ideal.